<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget {
      background-color: #F6F7FB;
      font-size: 11pt;
    }

    QGroupBox {
      background-color: #FFFFFF;
      border: 1px solid #D7DBE7;
      border-radius: 10px;
      margin-top: 18px;
      font-weight: 700;
    }

    QGroupBox QWidget {
      font-weight: 400;
    }

    QGroupBox::title {
      subcontrol-origin: margin;
      subcontrol-position: top left;
      left: 12px;
      padding: 0 10px;
      color: #111827;
      font-weight: 800;
      background-color: #FFFFFF;
    }

    QLabel#l_finishTitle {
      font-size: 14pt;
      font-weight: 700;
    }

    QLabel {
      color: #111827;
      font-weight: 400;
    }

    QLabel#l_installStatus {
      color: #4B5563;
    }

    QLineEdit,
    QTextBrowser {
      background-color: #FFFFFF;
      border: 1px solid #D7DBE7;
      border-radius: 8px;
      padding: 6px 8px;
    }

    QTextBrowser {
      font-family: &quot;Cascadia Mono&quot;, &quot;Consolas&quot;, &quot;Courier New&quot;;
    }

    QLineEdit:focus,
    QTextBrowser:focus {
      border: 1px solid #9AA7C7;
    }

    QPushButton {
      background-color: #FFFFFF;
      border: 1px solid #CFD6EA;
      border-radius: 8px;
      padding: 6px 14px;
    }

    QPushButton:hover {
      background-color: #F0F3FA;
    }

    QPushButton:pressed {
      background-color: #E6EBF7;
    }

    QPushButton:disabled {
      color: #9CA3AF;
      background-color: #F3F4F6;
      border-color: #E5E7EB;
    }

    QPushButton#b_next {
      background-color: #3B82F6;
      border: 1px solid #3B82F6;
      color: #FFFFFF;
      font-weight: 600;
    }

    QPushButton#b_next:hover {
      background-color: #2563EB;
      border-color: #2563EB;
    }

    QPushButton#b_next:pressed {
      background-color: #1D4ED8;
      border-color: #1D4ED8;
    }

    QPushButton#b_next:disabled {
      background-color: #93C5FD;
      border-color: #93C5FD;
      color: #FFFFFF;
    }

    QProgressBar {
      background-color: #FFFFFF;
      border: 1px solid #D7DBE7;
      border-radius: 8px;
      text-align: center;
    }

    QProgressBar::chunk {
      border-radius: 7px;
      background-color: #93C5FD;
    }

    QCheckBox {
      spacing: 8px;
    }

    QCheckBox::indicator {
      width: 16px;
      height: 16px;
    }

    QCheckBox::indicator:unchecked {
      background-color: #FFFFFF;
      border: 1px solid #CFD6EA;
      border-radius: 4px;
    }

    QCheckBox::indicator:checked {
      background-color: #93C5FD;
      border: 1px solid #93C5FD;
      border-radius: 4px;
    }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageWelcome">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="gb_programInfo">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="title">
             <string>Program Info</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="l_programName">
                 <property name="text">
                  <string>3DEngineViewer</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="l_programVersion">
                 <property name="text">
                  <string>Version 1.0</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="l_developer">
                   <property name="text">
                    <string>Developer: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="l_developerName">
                   <property name="text">
                    <string>Pavlo Kurishchenko</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="l_buildDate">
                   <property name="text">
                    <string>Build date: </string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="l_buildDateValue">
                   <property name="text">
                    <string>00.00.0000</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageLicense">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QGroupBox" name="gb_publicLicense">
            <property name="title">
             <string>Public License</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QTextBrowser" name="tb_licenseText"/>
               </item>
               <item>
                <widget class="QCheckBox" name="cb_acceptLicense">
                 <property name="text">
                  <string>I accept the terms of the license agreement</string>
                 </property>
                 <property name="tristate">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSerial">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QGroupBox" name="gb_serialVerification">
            <property name="title">
             <string>Serial Verification</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="l_serialNumber">
               <property name="text">
                <string>Enter serial number:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_serialNumber"/>
             </item>
             <item>
              <widget class="QLabel" name="l_email">
               <property name="text">
                <string>Email for installation report (optional):</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_email"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageDirectory">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QGroupBox" name="gb_choosePath">
            <property name="title">
             <string>Choose Path</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Enter install directory path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="le_installPath"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="b_browse">
                   <property name="text">
                    <string>Browse</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSettings">
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QGroupBox" name="gb_afterSettings">
            <property name="title">
             <string>Options after Installation</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="cb_desktopShortcut">
                 <property name="text">
                  <string>Create desktop shortcut</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="cb_startMenuShortcut">
                 <property name="text">
                  <string>Create Start Menu shortcut</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="cb_runAfterInstall">
                 <property name="text">
                  <string>Run application after installation</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Orientation::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageInstall">
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="QGroupBox" name="gb_installStatus">
            <property name="title">
             <string>Install Status</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_17">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="l_installing">
                   <property name="text">
                    <string>Installing...</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QProgressBar" name="pb_installProgress">
                   <property name="value">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="l_installStatus">
                 <property name="text">
                  <string>Preparing...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageFinish">
         <layout class="QVBoxLayout" name="verticalLayout_finish">
          <item>
           <widget class="QLabel" name="l_finishTitle">
            <property name="text">
             <string>Installation complete</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gb_finishDetails">
            <property name="title">
             <string>Details</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_finish">
             <item row="0" column="0">
              <widget class="QLabel" name="l_finishPath">
               <property name="text">
                <string>Install path:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="l_finishPathValue">
               <property name="text">
                <string>-</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="l_finishDate">
               <property name="text">
                <string>Install date:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="l_finishDateValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="l_finishTime">
               <property name="text">
                <string>Install time:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="l_finishTimeValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="l_finishSerial">
               <property name="text">
                <string>Serial number:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="l_finishSerialValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="l_finishEmail">
               <property name="text">
                <string>Email report to:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLabel" name="l_finishEmailValue">
               <property name="text">
                <string>-</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="l_finishDesktop">
               <property name="text">
                <string>Desktop shortcut:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="l_finishDesktopValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="l_finishStartMenu">
               <property name="text">
                <string>Start Menu shortcut:</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="l_finishStartMenuValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="l_finishRun">
               <property name="text">
                <string>Run after install:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLabel" name="l_finishRunValue">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="b_back">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_next">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
